### PR TITLE
Combine + simplify nav/masthead

### PR DESCRIPTION
**Pull request by tylersticka**
Now uses top-down drawer similar to [Solid](http://solid.buzzfeed.com/), with lighter colors that are mostly inherited by default:

pr

Does not yet address IE9 issues related to dependency on classList, but paves the way for that to be a much simpler change in a future PR.

---

Refs: #14

@saralohr @nicolemors

### CODE REVIEW

File: app/templates/src/assets/fabricator/styles/components/layout.css [hunk 1]
```diff
@@ -3,68 +3,30 @@
...
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    overflow-x: hidden;
+    overflow-y: auto;
+    box-sizing: border-box;
+    width: var(--Layout-nav-width);
+    height: 100%;
+    margin: 0;
```
Comment (erikjung) [inline app/templates/src/assets/fabricator/styles/components/layout.css]: Is this `margin` still needed here since it gets set above?

Comment (tylersticka) [inline app/templates/src/assets/fabricator/styles/components/layout.css]: Nope, good catch! I'll remove.

**Comment by erikjung** [comment]
Looks good overall. Only other nitpick from me is regarding the mix of `em` and `rem` used for margin and padding values. Can we make some custom properties for spacing and normalize the values a bit?

This is also something that can be deferred to another issue and addressed later. I assume there are similar opportunities for cleanup elsewhere. Thoughts?

**Comment by tylersticka** [comment]
@erikjung Sorry for the delay in responding, it's just a complicated question for me to answer.

I feel like I could rewrite and simplify the whole UI at this point, but the more changes I make the bigger pain it is to back-port those to this generator repository. This (mostly reductive) PR took two separate attempts and three separate instances to create (the generator, a local dev repo and a test instance of the generator output), and I spent a lot more time on it than I anticipated.

If tools existed for developing, viewing and pushing updates to the UI in a single location, I honestly think I could revise the whole thing in a couple of days. It's pretty frustrating as-is, hence my lack of enthusiasm for big, sweeping changes.

**Comment by erikjung** [comment]
:+1: LGTM. We can address other minor things later.

**Pull request merged**